examples/llm/icl_eval/evaluate_model.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2022 MosaicML Examples authors
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+import time
+
+import torch
+from composer.loggers import InMemoryLogger
+from composer.trainer import Trainer
+from model_loading import load_composer_model
+from omegaconf import DictConfig
+from omegaconf import OmegaConf as om
+
+from examples.common.builders import build_evaluators
+from examples.llm.src.tokenizer import TOKENIZER_REGISTRY
+
+
+def validate_cfg(eval_cfg):
+    assert eval_cfg.get('dataset_uri',
+                        None) is not None, 'dataset_uri is required!'
+    assert 'type' in eval_cfg
+    assert 'num_fewshot' in eval_cfg
+    assert 'batch_size' in eval_cfg
+    assert 'metrics' in eval_cfg
+    assert 'formatting_options' in eval_cfg
+    assert 'prompt_string' in eval_cfg.get('formatting_options')
+    assert 'example_delimiter' in eval_cfg.get('formatting_options')
+    assert 'continuation_delimiter' in eval_cfg.get('formatting_options')
+    assert 'label' in eval_cfg
+
+
+def get_evaluators_from_config(cfg):
+    tokenizer = TOKENIZER_REGISTRY[cfg.tokenizer.type](**cfg.tokenizer.args)
+    evaluators = []
+    logger_keys = []
+    for eval_cfg in cfg.icl_tasks:
+        validate_cfg(eval_cfg)
+        dataset_uri = eval_cfg.get('dataset_uri')
```

Comment:
since these args are non optional, should probably just use . notation like `eval_cfg.dataset_uri`, right?